packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -981,9 +980,9 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
           const abort = await this.registerDevice(FAILBACK_UTIL);
 
           if (!abort && !this.isDeviceRegistered()) {
-            const abort = await this.restorePreviousRegistration(FAILBACK_UTIL);
+            const abortNew = await this.restorePreviousRegistration(FAILBACK_UTIL);
```

Comment:
Was there any exception by eslint on this? (It seems fine to have same variable since there scope is in different blocks anyway)

Reply:
We are getting lint errors for this as per rule : @typescipt-eslint/no-shadow